pages/index.vue

```diff
@@ -1,60 +1,13 @@
+<template>
+  <TodoApp />
+</template>
 <script>
-import { mapMutations } from 'vuex'
+import TodoApp from '../components/todo/Index.vue'
 
 export default {
-  name: 'LandingPage',
-  data() {
-    return {
-      
-    }
-  },
-  computed: {
-    todos() {
-      return this.$store?.state?.todos?.list || []
-    }
+  name: 'Todo',
+  components: {
+    TodoApp,
```

Comment:
nitpicking: As the component works more like a wrapper than an app itself, we could name it like that.